### ISSUE TITLE
pagespeed breaks some images in Chrome on an iPhone

### ISSUE THREAD

**Issue by OHCSupport** [apache/incubator-pagespeed-mod#1340]
We have a website that does not properly show some images on some pages in Chrome on an iPhone. The same pages work fine in Safari on mobile. They work fine in Chrome on a Macbook, as they do in Safari and Firefox on a Macbook. Pages using the same template sometimes render images properly and sometimes don't. For instance, the logo is on every page but it sometimes doesn't render. I can't seem to isolate a pattern. Here is a page with broken images: http://observehiscommandments.org/turn-the-other-cheek/ Here is a page without broken images: http://observehiscommandments.org/prove-yourselves-cautious-as-serpents-and-yet-innocent-as-doves/ Needless to say, we are using the latest operating systems and browser versions in all cases. We recognized that "pagespeed" is in the url for the broken images. So we tried appending each of the following to the URL of the affected pages and in each case the problem went away:

?ModPagespeed=off
?ModPagespeed=on&ModPagespeedFilters=
?ModPagespeed=on&ModPagespeedFilters=extend_cache
?ModPagespeed=on&ModPagespeedFilters=combine_css
?ModPagespeed=on&ModPagespeedFilters=inline_css
?ModPagespeed=on&ModPagespeedFilters=inline_javascript
?ModPagespeed=on&ModPagespeedFilters=insert_image_dimensions
?ModPagespeed=on&ModPagespeedFilters=rewrite_images
?ModPagespeed=on&ModPagespeedFilters=rewrite_css
?ModPagespeed=on&ModPagespeedFilters=rewrite_javascript

**Comment by crowell** [comment]
what version of PageSpeed are you using? 

cloudflare seems to be stripping the headers.

**Comment by OHCSupport** [comment]
I am not aware of using it at all except it shows up in the URLs of images.
I presume this means it was part of the image that was used to create the
server. Either way, why do you think Cloudflare is stripping the headers? I
got the same results when I turned Cloudflare off.

**Comment by oschaaf** [comment]
It seems that today the response headers do indicate the mod_pagespeed version you are running: 1.9.32.3.

I think you are running into https://github.com/pagespeed/mod_pagespeed/issues/1256 which was fixed in release 1.9.32.13.
My advice would be to upgrade the module to the latest stable, but if you can't do that in the sort term @jmarantz  posted a workaround in configuration:

https://github.com/pagespeed/mod_pagespeed/issues/1256#issuecomment-176333705

[1] response headers from querying the index:

```
HTTP/1.1 200 OK
Date: Wed, 13 Jul 2016 10:24:37 GMT
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
X-Powered-By: PHP/5.6.18
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma: no-cache
Link: <http://observehiscommandments.org/wp-json/>; rel="https://api.w.org/", <http://wp.me/p6Y6bN-4w>; rel=shortlink
X-Frame-Options: SAMEORIGIN
Vary: Accept-Encoding
X-Mod-Pagespeed: 1.9.32.3-4523
Cache-Control: max-age=0, no-cache
Server: cloudflare-nginx
CF-RAY: 2c1c06b831f60c05-AMS
Content-Encoding: gzip
```
- edit: fix lost formatting -

**Comment by jeffkaufman** [comment]
It sounds like this is a duplicate of #1256.  If you still run into this after upgrading past 1.9.32.13, please reopen.
